change/@fluentui-react-datepicker-compat-29b1faa0-25ce-495b-ac31-4c60408a64e6.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "breaking: Remove componentRef since it's no longer necessary, fix ref forwarding, and fix focus movement when closing/opening popup.",
```

Comment:
What is the status of `DatePickerCompat`? Are we still able to make breakin changes?

Reply:
Yes, since we are in `0.x.x` we can. We should try not to, but I think this is a good enough change to push one.

Reply:
Just one follow up question: could this be done over two releases?

Ex:
1. Deprecate the old functionality in this release
2. Introduce the new functionality
3. Remove the deprecated functionality in the next release

I know this is a bit more work on our end but I feel it's good practice, even though from a semver pov it's fine to break, as it gives users a warning that the break is coming.

Reply:
makes sense, I'll keep doing it if needed for future breaks as well. Thanks!

Reply:
After talking offline, we will make a one-off exception for this case and follow the deprecation flow from now on. This decision was made mainly because of the ref change needing a hard break so deprecating would mean releasing 0.2.0 and then releasing 0.3.0 to remove the deprecation. Note that since we follow `0.X.X` this hard break is allowed and since useDatePicker__unstable is marked as unstable, breaks are expected as well.